csrankings.csv

```diff
@@ -1,4 +1,4 @@
-name,affiliation,homepage,scholarid
+﻿name,affiliation,homepage,scholarid

```

Comment:
Please delete this diff, which is spurious.

Reply:
Mayank Singh 0001,IIT Gandhinagar,http://www.iitgn.ac.in/faculty/comp/mayank.htm,FOjZrQcAAAAJ
Nipun Batra 0001,IIT Gandhinagar,https://www.iitgn.ac.in/faculty/comp/nipun.htm,rFGzHlIAAAAJ

I have only added this 2 lines in the csv file.  Could you please elaborate a little which is the difference that you want me to delete.

Reply:
You need to get rid of the first line (name,affiliation, etc.)

Reply:
was not able to delete the mentioned 1st line. So created one new pull request. Please accept that. 
https://github.com/emeryberger/CSrankings/pull/1302  , Thank You.